src/pages/QuestionPage.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import QuestionBox from "../components/ui/QuestionBox";
+
+const QuestionPage = () => {
+  return (
+    <div>
```

Comment:
semantically it should be main not div. It will be the main element of the page (once we do the page)

Reply:
THX ,  
![image](https://github.com/user-attachments/assets/dfd6e0f1-6a6d-4aee-b3c0-7f058a7fbcfb)